pymc3/tests/test_gp.py

```diff
@@ -203,6 +279,14 @@ def warp_func(x, a, b, c):
             cov = gp.cov.WarpedInput(1, warp_func=warp_func, args=(1,10,1), cov_func=cov_m52)
         K = theano.function([], cov(X))()
         self.assertAlmostEqual(K[0,1], 0.79593, 4)
+        K = theano.function([], cov(X,X))()
+        self.assertAlmostEqual(K[0,1], 0.79593, 4)
+
+    def test_raises(self):
+        cov_m52 = gp.cov.Matern52(1, 0.2)
+        self.assertRaises(AssertionError, gp.cov.WarpedInput, 1, cov_m52, "not callable")
```

Comment:
thanks for including tests for bad input!  this is a nitpick, but I find it much more readable to use the context manager here:

    with self.assertRaises(AssertionError):
        gp.cov.WarpedInput(1, cov_m52, 'not callable')